lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3093,6 +2354,12 @@ def model_add_daylighting_controls(model)
     return true
   end
 
+  # For backward compatibility, infiltration standard not used for 2013 and earlier
+   # @return [Bool] true if successful, false if not
+  def baseline_apply_infiltration_standard(model, climate_zone)
```

Comment:
To be consistent with other methods, the name should probably be `model_baseline_apply_infiltration_standard` since it's under `Standards.Model.rb`

Reply:
We might also want to return `true` to be consistent with the YARD comments.

Reply:
OK